.github/workflows/k8s-generated.yaml

```diff
@@ -0,0 +1,29 @@
+name: K8S generated file check
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+jobs:
+  generate-k8s-api:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.2
+      - name: Checkout code
+        uses: actions/checkout@v1
+        with:
+          path: ./src/github.com/${{ github.repository }}
+      - name: Install protobuf dependencies
+        run: |
+          sudo apt install -y protobuf-compiler libprotobuf-dev
+      - name: Check generated files
+        env:
+          GOPATH: ${{ runner.workspace }}
+        run: |
+          make generate-k8s-api
+          git diff --exit-code
```

Comment:
We're having more and more GitHub Actions, we should start thinking about grouping them. Maybe we could add this job to the go-mod workflow? Does that make sense?

Reply:
yup totally making sense to me :), so let me group all go related action together.